content/en/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -0,0 +1,73 @@
+---
+title: API for sidecar containers
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+Sidecar containers are the secondary containers that runs along with the main
+application container within the same {{< glossary_tooltip text="Pod" term_id="pod" >}}.
+These containers are used to enhance or extend the functionality of the primary container
+by providing additional services, or functionality such as logging, monitoring, security,
+or data synchronization, without directly altering the primary application code.
+
+<!-- body -->
+
+Starting with Kubernetes 1.28 in alpha, a feature gate named `SidecarContainers`
+allows you to specify a `restartPolicy` for containers listed with a Pod's `initContainers`
+field. These restartable _sidecar_ containers are independent of other
+[init containers](/docs/concepts/workloads/pods/init-containers/) and main application
+container within the same pod. These can be started, stopped, or restarted independently
+of the main container and other init containers.
+Container [probes](/docs/concepts/workloads/pods/pod-lifecycle/#types-of-probe)
+can also be added to control their lifecycle.
+
+If an init container is created with its `restartPolicy` set to `Always`, it will
+start and remain running during the entire life of the Pod, which is useful for
+running supporting services separated from the main application containers.
+
+If a `readinessProbe` is specified for this init container, its result will be used
+to determine the `ready` state of the Pod.
```

Comment:
If you want to talk about "probes", then consolidate those words into one or two paragraphs. Please avoid jumping between (sub)topics back and forth.